app/code/Magento/Checkout/Block/Cart/Shipping.php

```diff
@@ -21,26 +21,35 @@ class Shipping extends \Magento\Checkout\Block\Cart\AbstractCart
     protected $layoutProcessors;
 
     /**
+     * @var \Magento\Framework\Serialize\Serializer\Json
+     */
+    private $serializer;
+
+    /**
      * @param \Magento\Framework\View\Element\Template\Context $context
      * @param \Magento\Customer\Model\Session $customerSession
      * @param \Magento\Checkout\Model\Session $checkoutSession
      * @param \Magento\Checkout\Model\CompositeConfigProvider $configProvider
      * @param array $layoutProcessors
+     * @param \Magento\Framework\Serialize\Serializer\Json|null $serializer
      * @param array $data
-     * @codeCoverageIgnore
+     * @throws \RuntimeException
      */
     public function __construct(
         \Magento\Framework\View\Element\Template\Context $context,
         \Magento\Customer\Model\Session $customerSession,
         \Magento\Checkout\Model\Session $checkoutSession,
         \Magento\Checkout\Model\CompositeConfigProvider $configProvider,
         array $layoutProcessors = [],
+        \Magento\Framework\Serialize\Serializer\Json $serializer = null,
```

Comment:
I think if you are adding a new optional param like this it should come last as you are trying to avoid breaking anything.